src/routes/mayan/MayanRouteBase.ts

```diff
@@ -82,62 +82,36 @@ import {
   toMayanChainName,
   txStatusToReceipt,
 } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-namespace
-export namespace MayanRoute {
-  export type Options = {
-    gasDrop: number;
-    slippageBps: number | 'auto';
-    optimizeFor: 'cost' | 'speed';
-  };
-  export type NormalizedParams = {
-    slippageBps: number | 'auto';
-  };
-  export interface ValidatedParams
-    extends routes.ValidatedTransferParams<Options> {
-    normalizedParams: NormalizedParams;
-  }
-}
-
-type Op = MayanRoute.Options;
```

Comment:
Types moved to types.ts and renamed to what they represent instead of ambiguous alphabets